src/sign/cosmos.tf

```diff
@@ -0,0 +1,30 @@
+module "cosmosdb_account" {
+  source                           = "git::https://github.com/pagopa/azurerm.git//cosmosdb_account?ref=v2.13.1"
+  name                             = "${local.project}-cosmos"
+  subnet_id                        = module.io_sign_snet.id
```

Comment:
for cosmos db u need to enable privet endpoints.
here an example
https://github.com/pagopa/selfcare-infra/blob/main/src/core/mongodb.tf#L38


for IO there is a subnet named `pendpoints` and u need to refer it as an external resource.
also u need to refer the private dns zone for cosmos db with as an external resource.